datadog/api/http_client.py

```diff
@@ -18,7 +18,7 @@
     import requests
     import requests.adapters
 except ImportError:
-    requests = None
+    requests = None  # type: ignore
```

Comment:
This is one of the recommended patterns for fixing Mypy errors when setting a missing module to `None`.